lib/config/executor/container_vm.go

```diff
@@ -64,7 +64,7 @@ type ExecutorConfigCommon struct {
 	ID string `vic:"0.1" scope:"read-only" key:"id"`
 
 	// Convenience field to record a human readable name
-	Name string `vic:"0.1" scope:"hidden" key:"name"`
+	Name string `vic:"0.1" scope:"read-only" key:"name"`
```

Comment:
any story on this change? I remember we made it hidden intentionally for container rename, and to avoid that mess up other attributes, @chenchenw-vmware copied this structure